client/sidebar/footer/VoiceController.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+	SidebarItem,
+	SidebarItemContent,
+	// SidebarItemAvatar,
+	SidebarItemSubtitle,
+	SidebarItemWrapper,
+	SidebarItemTitle,
+	SidebarItemContainer,
+	SidebarItemActions,
+	SidebarItemAction,
+	SidebarFooter,
+	SidebarSection,
+	SidebarSectionTitle,
+	// TopBarAction,
+} from '@rocket.chat/fuselage';
+import React, { FC, ReactElement } from 'react';
+
+import { useCallActions, useCallState } from '../../contexts/CallContext';
+// import { useAvatarTemplate } from '../hooks/useAvatarTemplate';
+
+const VoiceController: FC = (): ReactElement | null => {
+	const call = useCallState();
+
+	const actions = useCallActions();
+
+	if (call.state !== 'IN_CALL' && call.state !== 'OFFER_RECEIVED') {
+		return null;
+	}
+
+	const subtitle = call.state === 'IN_CALL' ? 'Call in Progress' : 'Calling';
+
+	return (
```

Comment:
tons of comments on this return...

Reply:
You can ignore most of the comments on this file, we are waiting for some functionality from the backend